assets/src/edit-story/components/panels/animation/effectChooser.js

```diff
@@ -119,21 +131,182 @@ const GridItemHalfRow = styled(GridItem)`
   grid-column-start: span 2;
 `;
 
+/**
+ * Because the effect chooser is hard coded these two effects lists help keep track of the current index
+ * current index is how we track focus for up and down arrows and setting active list item when menu is opened.
+ * These values map to the value prop and then if a direction is present,
+ * it's added to the name in order of appearance in the dropdown to track index properly and uniquely
+ */
+
+const FOREGROUND_EFFECTS_LIST = [
+  'No Effect',
+  'Drop',
+  'Fade In',
+  `Fly In ${DIRECTION.LEFT_TO_RIGHT}`,
+  `Fly In ${DIRECTION.TOP_TO_BOTTOM}`,
+  `Fly In ${DIRECTION.RIGHT_TO_LEFT}`,
+  `Fly In ${DIRECTION.BOTTOM_TO_TOP}`,
+  'Pulse',
+  `Rotate In ${DIRECTION.LEFT_TO_RIGHT}`,
+  `Rotate In ${DIRECTION.RIGHT_TO_LEFT}`,
+  'Twirl In',
+  `Whoosh In ${DIRECTION.LEFT_TO_RIGHT}`,
```

Comment:
I know these are internal values, but localized values will not break the comparison with these right?

Reply:
oh heck! great catch, I hadn't registered that the title the dropdown had previously was the one getting translated 🙀  - updated to pass in the animation _type_ from the panel and that is now the `value` getting sent to the effectChooser. Bright side too is I got to nix all those strings in favor of the constants in the animations directory.